extensions/wikia/CommunityPage/styles/components/_rail.scss

```diff
@@ -0,0 +1,66 @@
+@import 'skins/shared/color';
+@import 'skins/oasis/css/core/breakpoints-variables';
+@import 'variables';
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/base/colors';
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/base/typography';
+
+$community-page-rail-background-color: mix($color-page, #000, 96%);
+@if $is-dark-wiki {
+	$community-page-rail-background-color: mix($color-page, $color-white, 96%);
+}
+$community-page-rail-width: 266px;
+$community-page-rail-width-small: 256px;
+$community-page-rail-module-width: 210px;
+$community-page-rail-module-spacing: 35px;
+$community-page-rail-spacing-small: 14px;
+$community-page-rail-spacing: 16px;
+$community-page-rail-spacing-large: 18px;
+$community-page-rail-spacing-xlarge: 20px;
+
+.community-page-rail {
+	background-color: $community-page-rail-background-color;
+	display: inline-block;
+	float: right;
+	padding: $community-page-rail-spacing-xlarge * 4 0 $community-page-rail-spacing-xlarge;
```

Comment:
I would separate this into 3 properties - it's harder to read like this especially with variables